proposals/using.md

```diff
@@ -0,0 +1,164 @@
+# using patterns and locals
+
+## Summary
+
+The language will add two need capabilities around the `using` statement in order to make resource
```

Comment:
Typo: need -> needed

Reply:
Why couple this two unrelated proposals together?

Reply:
This is about improving the usability of the `using` statement hence grouping all aspects of that together.